.github/workflows/chromatic.yml

```diff
@@ -1,10 +1,15 @@
 name: "Chromatic"
 
-on: push
+on:
+  workflow_dispatch:
```

Comment:
Do we also want this to run on merges to `main`?

Reply:
Oh yes, thank you!

Reply:
90f08eea5d073462592ee65daba2ec606d875daf

Reply:
🥳